hudi-common/src/test/java/org/apache/hudi/common/util/TestFSUtils.java

```diff
@@ -43,11 +49,33 @@
 import java.util.stream.Stream;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 
 /**
  * Tests file system utils.
  */
 public class TestFSUtils extends HoodieCommonTestHarness {
+  private long minRollbackToKeep = 10;
+  private long minCleanToKeep = 10;
+  protected transient FileSystem fs;
+  protected String basePath = null;
+
+  @Before
```

Comment:
You don't need this init, the HoodieTestHarness actually creates a DFS based file system which works for log records and should not run into the exception for which you might be creating the local file system

Reply:
incorporated the comments.